### PR TITLE
132 award logo on other sites

### PR DESCRIPTION
**Pull request by Nikomus**
- Generalises styling from C-App portal to ESA and PIP pages.
- Better support for particularly large/small screen sizes.

### CODE REVIEW

File: src/css/style.css [hunk 1]
```diff
@@ -1151,6 +1148,22 @@ footer a {
...
+}
+
+.footer-container:nth-child(1) {
+  line-height: 1em;
+  margin-bottom: 1em;
+  max-width: 40%;
+}
+
+footer {
+  padding: 0 1.5em 0em 1.5em;
```
Comment (katjam) [inline src/css/style.css]: Could be `padding: 0 1.5em;`

File: src/css/style.css [hunk 2]
```diff
@@ -1175,8 +1188,44 @@ footer #privacy {
...
+  margin-right: 2em;
+}
+
+.sponsor-images {
+  margin-bottom: 1rem;
+  text-align: right;
+  padding: 0;
+}
+
+.sponsor-text {
```
Comment (katjam) [inline src/css/style.css]: repeat of 1197

File: src/css/style.css [hunk 3]
```diff
@@ -1630,56 +1658,62 @@ footer #feedback {
...
   }
 
-  .c-app .privacy-link {
+  .privacy-link {
     align-self: flex-end;
     text-align: left;
   }
   
-  .c-app footer {
+  .privacy-link {
```
Comment (katjam) [inline src/css/style.css]: repeat of 1661?

File: src/css/style.css [hunk 4]
```diff
@@ -1630,56 +1658,62 @@ footer #feedback {
...
   
   /* "Brought to you by" block */
-  .c-app .footer-container:nth-child(3) {
+  .footer-container:nth-child(3) {
     margin: 0;
     flex-basis: 70%;
   }
   
-  .c-app .sponsor-images {
+  .sponsor-images {
```
Comment (katjam) [inline src/css/style.css]: This is repeat of 1214?

File: src/css/style.css [hunk 5]
```diff
@@ -1630,56 +1658,62 @@ footer #feedback {
...
   margin-bottom: 1rem;
   text-align: right;
 }
   
-  .c-app .sponsor-text {
+  .sponsor-text {
```
Comment (katjam) [inline src/css/style.css]: Also repeat?

File: src/css/style.css [hunk 6]
```diff
@@ -1630,56 +1658,62 @@ footer #feedback {
...
     text-align: right;
   }
   
   .social-profile-buttons {
     margin: 2em 0 0 1.5em;
   }
     
-  .c-app .social-profile-buttons {
+  .social-profile-buttons {
     margin: 2em 0 0 0;
```
Comment (katjam) [inline src/css/style.css]: different at 1682/1713

File: src/css/style.css (hunk 3 above)
Comment (Nikomus) [reply src/css/style.css]: Not sure what happened here 🙃

**Pull request merged**